src/main/java/com/db/crud/voting/service/impl/AgendaServiceImpl.java

```diff
@@ -146,11 +161,11 @@ private void verifyUserVoted(User user, List<User> usersVoted) {
         }
     }
 
-    private void sortVote(String vote, Agenda agenda) {
-        if (vote.equals("Y")) {
+    private void sortVote(Vote vote, Agenda agenda) {
```

Comment:
O uso do prefixo sort no nome deste método pode gerar confusão. Normalmente, sort está associado a métodos ou algoritmos de ordenação de dados. Ao ler apenas o nome do método, pode-se ter a impressão de que ele realiza uma ordenação ou algum tipo de priorização de votos, o que não é o caso. Recomendo encontrar um nome mais significativo e alinhado ao que o método realmente faz, como por exemplo processVote.

Reply:
Não tinha visto dessa forma e sim, acaba causando uma confusão. Nome mudado para "countVote" (o método justamente filtra a informação e contabiliza os votos) no commit [refactor: Rename method that handle Votes](https://github.com/AurorinhaBoreal/VotacaoCRUD-BackEnd/pull/37/commits/f33fb63fec4bee9e492d5737240eac7e148a9327)